rep-0008.rst

```diff
@@ -67,12 +133,12 @@ Module with msgs/srvs::
           |- yourfiles.py
      |- scripts/
         |- non-exported python files
-    
+
 We distinguish between "nodes" and "scripts" for clarity to
 users. Nodes are executable Python files that conform to the ROS node
 API. Scripts are executable Python files that do not conform to the
 ROS node API.
```

Comment:
Sure, we want the REP to reflect best practices. Documentation should be updated where appropriate as time is available.

Some of the how-to documentation I wrote recommended putting executable Python files in ``scripts`` because I did not know about ``nodes``.  It's not exactly wrong, but that would have been a good place to mention the other option.